src/main/java/com/wavefront/sdk/entities/metrics/WavefrontMetricSender.java

```diff
@@ -31,6 +31,15 @@
   void sendMetric(String name, double value, @Nullable Long timestamp, @Nullable String source,
                   @Nullable Map<String, String> tags) throws IOException;
 
+  /**
+   * Similar to {@link #sendMetric(String, double, Long, String, Map)}, only the {@code point}
+   * argument is expected to already be in Wavefront Data Format
+   *
+   * @param point a single metric, encoded in Wavefront Data Format
+   * @throws IOException if there was an error sending the metric.
+   */
+  void sendMetric(String point) throws IOException;
```

Comment:
can you rename this to 
sendFormattedMetric()
to avoid confusion with sendMetric() method?

Reply:
Done